Gaijinizer patch/Map162.txt

```diff
@@ -343,9 +347,9 @@ There was data for mail forwarding ...#MTLed
  メンバーコードが記載されていたよ…
 > CONTEXT: Map162/events/2/pages/1/84/Dialogue/yuuma
 > CONTEXT: Map162/events/2/pages/2/70/Dialogue/yuuma
-I did not know the forwarding address was garbled
-but...In the Sender field, her M.O.W.
-The member code was listed ...#MTLed
+The address it'd been forwarded to was garbled,
+but her M.O.W member ID was displayed in the
+sender field...#Ron
 > END STRING
```

Comment:
Minor wording change:
>_**The address it'd been forwarded to had been
corrupted, but her M.O.W member ID was in the
sender field..**_